source/release-notes/2.9.1.rst

```diff
@@ -0,0 +1,46 @@
+.. _PMM-2.9.1:
+
+================================================================================
+*Percona Monitoring and Management* 2.9.1
+================================================================================
+
+:Date: August 3, 2020
+:Installation: `Installing Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/2.x/install/index-server.html>`_
+
+*PMM* (`Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/index.html>`_)
+is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL
+performance.
+
+Improvements
+================================================================================
+
+* :jirabug:`PMM-6230`: Home Dashboard: Customizations retained across updates
+* :jirabug:`PMM-6300`: Query Analytics Dashboard: Column sorting arrows made easier to use (Thanks to user debug for reporting this issue)
+* :jirabug:`PMM-6208`: Security Threat Tool: Silence viewed but unactioned alerts
+* :jirabug:`PMM-6315`: Query Analytics Dashboard: Improved metrics names and descriptions
+* :jirabug:`PMM-6274`: MySQL User Details Dashboard: View selected user's queries in Query Analytics Dashboard
+* :jirabug:`PMM-6266`: Query Analytics Dashboard: Pagination device menu lists 25, 50 or 100 items per page
+* :jirabug:`PMM-6262`: PostgreSQL Instance Summary Dashboard: Descriptions for all 'Temp Files' views
+* :jirabug:`PMM-6253`: Query Analytics Dashboard: Improved SQL formatting in Examples panel
+* :jirabug:`PMM-6211`: Query Analytics Dashboard: Loading activity spinner for Example, Explain and Tables tabs
+* :jirabug:`PMM-6162`: Consistent sort order in dashboard drop-down filter lists
+* :jirabug:`PMM-5132`: Better message when filter search returns nothing
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PMM-5783`: Bulk failure of SHOW ALL SLAVES STATUS scraping on PS/MySQL distributions triggers errors
```

Comment:
.. was causing a lot of errors in logs and alerting in different systems

Reply:
That's already said. You want to add 'which fills up logs and affects other systems'?

Reply:
sorry, lost  the next  line . no, it's OK